server/cluster/cluster.go

```diff
@@ -404,7 +404,7 @@ func (c *RaftCluster) Start(s Server, bootstrap bool) (err error) {
 }
 
 func (c *RaftCluster) checkSchedulingService() {
-	if c.isPDServiceMode {
+	if c.isKeyspaceGroupEnabled {
```

Comment:
If we only use pd service and scheduling service, is there keyspace group? Keyspace group similiar be about tso only.

Reply:
see https://github.com/tikv/pd/pull/8984/commits/2e8acc2a95e51cebc76dd9b59806132143ac8801